src/lazy/expanded/macro_evaluator.rs

```diff
@@ -2549,6 +2687,107 @@ mod tests {
         stream_eq(e_expression, r#" [1, 2, 3, 4, 5, 6, 7] "#)
     }
 
+    #[test]
+    fn default_eexp() -> IonResult<()> {
+        stream_eq(
+            r#"
+            (:add_macros
+                (macro foo (x?)
+                    (.make_string "Hello, " (.default (%x) "World!"))))
+            (:foo "Gary")
+            (:foo)
+        "#,
+            r#"
+            "Hello, Gary"
+            "Hello, World!"
```

Comment:
After reading the next two tests I feel compelled:

```suggestion
                    (.make_string "Hello, " (.default (%x) "world!"))))
            (:foo "Gary")
            (:foo)
        "#,
            r#"
            "Hello, Gary"
            "Hello, world!"
```